orchagent/orchdaemon.cpp

```diff
@@ -629,9 +630,12 @@ bool OrchDaemon::init()
 
     m_orchList.push_back(&CounterCheckOrch::getInstance(m_configDb));
 
-    vector<string> p4rt_tables = {APP_P4RT_TABLE_NAME};
-    gP4Orch = new P4Orch(m_applDb, p4rt_tables, vrf_orch, gCoppOrch);
-    m_orchList.push_back(gP4Orch);
+    if (gMySwitchType != "dpu")
```

Comment:
Is this applicable? This should not be impacting even if initialized, correct?

Reply:
After reworking of P4Orch this is not needed anymore.